exercises/resistor-color-duo/.meta/src/reference/java/ResistorColorDuo.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Example Class
+ */
+public class ResistorColorDuo {
```

Comment:
Please change the access modifier to package private.

This applies to all used access modifiers: try to be as restrictive as possible.